plugin.video.cbc-sports/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.cbc-sports" name="CBC Sports" version="1.0.1" provider-name="Jbinkley60">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.beautifulsoup4" version="4.3.2"/>
+    <import addon="script.module.html5lib" version="0.999.0"/> 
+    <import addon="script.module.requests" version="2.9.1"/> 
+    </requires>
+    <extension point="xbmc.python.pluginsource"
+               library="addon.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <license>GPL 2.0 or Later</license>
+        <assets>
+        <icon>/resources/icon.png</icon>
+        <fanart>/resources/fanart.jpg</fanart>
+        </assets>
+        <summary lang="en_GB">Live Events, Replays, and Highlights from CBC Sports</summary>
```

Comment:
Addon.xml missed source tag to point to your upstream repo

Reply:
Fixed in addon.xml file